Getting-Started/Backoffice/Login/index.md

```diff
@@ -1,4 +1,12 @@
 # Login screen
+To access the backoffice, you will need to login. You can do this by adding `/umbraco` to the end of your website address e.g. http://www.myUmbracoWebsite.com/umbraco
```

Comment:
Just a stupid question.  Is www.myUmbracoWebsite.com not difficult to understand?  E.g. http://mywebsite.com/umbraco makes more sense IMHO.  Otherwise you would think that you need to add umbraco in your hostname.

Or is this too far fetched?

Reply:
That's a fair point. I was thinking of test.com and other domains. mywebsite works.